mesh/nurbs.hpp

```diff
@@ -95,18 +95,23 @@ class NURBSPatch
 
    Array<KnotVector *> kv;
 
-   int sd, nd;
-
-   void swap(NURBSPatch *np);
-
    // Special B-NET access functions
+   //  - SetLoopDirection(int dir) flattens the multi-dimensional B-NET in the
+   //    requested direction. It effectively creates a 1D net.
+   //  - The slice(int, int) operator is the access function in that flattened structure.
+   //    The first int gives the slice and the second int the element in that slice.
+   //  - Both routines are used in 'InsertKnot', 'DegreeElevate' and 'UniformRefinement'.
+   //  - In older implementations slice(int int) was implemented as operator()(int, int)
+   int nd; // Number of knots in flattened structure
+   int ls; // Number of variables per knot in flattaned structure
```

Comment:
Thanks for the documentation. One typo: `flattaned` -> `flattened`

Reply:
Thanks.